docs/howto/operate/ci-cd.md

```diff
@@ -0,0 +1,138 @@
+# Our CI/CD system
+
+The best place to look to learn about the latest state of our *automatic* hub deployment
+is to look at [the `deploy-hubs.yaml` GitHub workflow](https://github.com/2i2c-org/pilot-hubs/blob/master/.github/workflows/deploy-hubs.yaml).
+That workflow defines the automatic hub deployment for many of our major clusters.
+
+:::{note}
+This is essentially a placeholder, it needs to be extended to talk a LOT more about our
+CI/CD system.
+:::
+
+## AWS cluster/hubs specific requirements
+
```

Comment:
Could we add a note about when these steps need to be followed? Is it for every single AWS hub? Only for hubs on new clusters?

If it's for every single AWS hub, I'd be include to add the "step by step" stuff to our "Add a new hub" guide.

Reply:
This is a cluster setup. Things could be a little bit confusing because we currently have a one-to-one hub:cluster mapping on all of our AWS communities (in fact, it is a 2:1 mapping since we deploy `staging` and `prod` on each cluster). I changed the title of the section to make clear this is a "cluster" setup.